Makefile

```diff
@@ -3,64 +3,64 @@
 all: tools build test
 
 up:
-	bin/up
+	bin/up.sh
```

Comment:
I'd rather not change that. Upstream projects also omit the extension: https://github.com/cloudfoundry/bosh-cli/tree/master/bin
It's also common for scripts in bin folders on Linux. Any reason to change it?

Reply:
I see. Bazel got confused with `bin/build`, thinking that it is a BUILD file. I'll see what I can do to prevent that other than changing the name of the files.